src/main/java/com/hackclub/hccore/commands/LocCommand.java

```diff
@@ -89,6 +88,29 @@ public boolean onCommand(CommandSender sender, Command cmd, String alias, String
                 }
                 break;
             }
+            // /loc rename <old name> <new name>
+            case "rename": {
+                if (args.length < 3) {
+                    sender.sendMessage(
+                            ChatColor.RED + "You must include both a location and a new name");
+                    break;
+                }
+                String newName = String.join("_", Arrays.copyOfRange(args, 2, args.length));
+                String oldName = args[1];
```

Comment:
Put `oldName` before `newName`

Reply:
which is why you should just return false